### PR TITLE
Document container and image garbage collection

### PR DESCRIPTION
**Pull request by csrwng**

### CODE REVIEW

**Comment by csrwng** [comment]
@derekwaynecarr @adellape ptal

File: admin_guide/garbage_collection.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+----
+====
+
+
+[[image-garbage-collection]]
+
+=== Image Garbage Collection
+
+Image garbage collection relies on disk usage as reported by cAdvisor on the node to decide
+which images to remove from the node. It takes the following settings into cosideration:
```
Comment (derekwaynecarr) [inline admin_guide/garbage_collection.adoc]: consideration

**Comment by derekwaynecarr** [comment]
one typo, then lgtm

**Comment by csrwng** [comment]
thx, fixed

**Comment by derekwaynecarr** [comment]
LGTM

File: admin_guide/garbage_collection.adoc [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+----
+====
+
+
+[[image-garbage-collection]]
+
+=== Image Garbage Collection
+
+Image garbage collection relies on disk usage as reported by cAdvisor on the node to decide
+which images to remove from the node. It takes the following settings into consideration:
```
Comment (ncdc) [inline admin_guide/garbage_collection.adoc]: Can we make it a bit clearer that image GC occurs when you're running out of free space for images?

Comment (ncdc) [inline admin_guide/garbage_collection.adoc]: Please add that image GC runs every 5 minutes, and is not configurable (the hardcoding is here: https://github.com/openshift/origin/blob/master/Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubelet/image_manager.go#L126)

Comment (csrwng) [inline admin_guide/garbage_collection.adoc]: Will do

**Comment by adellape** [comment]
@csrwng This needs a rebase due to some config topics being moved from `admin_guide` up into the `install_config` book. I think this topic could also get moved into Install & Config, come to think of it. If it does, then the links to `master_node_configuration.html` should still be fine.

**Comment by csrwng** [comment]
Yup, will update today

**Pull request closed without merging**